core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -41,7 +41,7 @@ object ConfigCommand {
     val opts = new ConfigCommandOptions(args)
 
     if(args.length == 0)
-      CommandLineUtils.printUsageAndDie(opts.parser, "Add/Remove entity (topics/clients) configs")
+      CommandLineUtils.printUsageAndDie(opts.parser, "Add/Remove entity (topics/clients/broker) configs")
```

Comment:
Good point. I've changed to "Add/Remove entity config for a topic, client or broker" (command only lets you change one at a time)